### PR TITLE
wired up account backend; added gitignore; added expert log_in and sign_up page

### PR DESCRIPTION
**Pull request by LiamFengLin**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,40 @@
+*.rbc
```
Comment (LiamFengLin) [inline .gitignore:1]: we can use this gitignore from now on since some stuff should be pushed to github

File: Gemfile.lock [hunk 2]
```diff
@@ -1,200 +0,0 @@
-GEM
```
Comment (LiamFengLin) [inline Gemfile.lock:1]: it is best that gemfile.lock is not pushed to github

File: app/assets/stylesheets/foundation_and_overrides.scss [hunk 3]
```diff
@@ -0,0 +1,1447 @@
+// Foundation by ZURB
```
Comment (LiamFengLin) [inline app/assets/stylesheets/foundation_and_overrides.scss:1]: external file. no need to look at this

File: app/views/users/confirmations/new.html.erb [hunk 4]
```diff
@@ -0,0 +1,16 @@
+<h2>Resend confirmation instructions</h2>
```
Comment (LiamFengLin) [inline app/views/users/confirmations/new.html.erb:1]: autogenerated

File: app/views/users/registrations/edit.html.erb [hunk 5]
```diff
@@ -0,0 +1,54 @@
+<h2>Edit <%= resource_name.to_s.humanize %></h2>
```
Comment (LiamFengLin) [inline app/views/users/registrations/edit.html.erb:1]: this is according to our design.

File: app/views/users/registrations/new.html.erb [hunk 6]
```diff
@@ -0,0 +1,76 @@
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name), :html => {:class => "registration"}) do |f| %>
```
Comment (LiamFengLin) [inline app/views/users/registrations/new.html.erb:1]: this is according to our design.

File: app/views/users/sessions/new.html.erb [hunk 7]
```diff
@@ -0,0 +1,52 @@
+<%= form_for(resource, as: resource_name, url: session_path(resource_name), :html => {:class => "registration"}) do |f| %>
```
Comment (LiamFengLin) [inline app/views/users/sessions/new.html.erb:1]: our design.

**Comment by bicro** [comment]
looks good

**Pull request merged**